ProfilerGUI/Source/Shared/ProfilerConfiguration.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using ProfilerGUI.Source.Configurator;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace ProfilerGUI.Source.Shared
+{
+    /// <summary> Collects all options needed to run the profiler. </summary>
+    public class ProfilerConfiguration
+    {
+        /// <summary>
+        /// The name of the config file.
+        /// </summary>
+        private const string ConfigFileName = "ProfilerGUI.json";
+
+        /// <summary>
+        /// Path to the config file.
+        /// </summary>
+        public static readonly string ConfigFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ConfigFileName);
```

Comment:
Again, we write to this file, hence I'd place in APP_DATA. I'm also pretty sure that we might need more than one config saveable.

Reply:
Actually it would be nice to have one config file for uploader and daemon

Reply:
Any reason we we're not re-using the already existing config file for the profiler?

Reply:
the existing config file is very simplistic on purpose (properties-like) to be able to parse it easily in C++. The uploader is a separate process and I didn't want to shoehorn the much more complicated config into this simple format. And also I didn't want to parse JSON in C++ for obvious reasons :P 

As to APP_DATA, see #29